db/documents.py

```diff
@@ -49,6 +72,8 @@ def __str__(self):
 
 document_type = sqla.Enum('oral', 'written', 'oral reexam', name='document_type', inherit_schema=True)
 
+class PaymentState:
+    NOT_ELIGIBLE, ELIGIBLE, DISBURSED = range(3)
```

Comment:
The distinction is an leftover of an early draft. I thought about changing those fields to bool (eligible / not eligible), but hesitated because of an possible feature in the future. Though i am not entirely sure, if this information is needed there at all. I'll put this in an local branch and change them to bools for now :)